signer/src/api/new_block.rs

```diff
@@ -369,6 +416,8 @@ async fn handle_withdrawal_reject(
         .write_withdrawal_reject_event(&event)
         .await?;
 
+    tracing::info!("handled withdrawal rejection event");
```

Comment:
This is fine, but we may want to consider using the "proper" event names here, and maybe even have them as a separate key, like "event-topic" or something. And by proper event names I mean the ones in https://github.com/stacks-network/sbtc/blob/6b7e313dc79313ad99949f5e374ba5804334aa8a/sbtc/src/events.rs#L170-L174

Reply:
Yeah good point, updated in [fa00743](https://github.com/stacks-network/sbtc/pull/1203/commits/fa00743c54a613f750d1c0b336ec67013240a0f3)